modules/library/src/test/scala/zio/elasticsearch/ElasticQuerySpec.scala

```diff
@@ -2042,6 +2043,84 @@ object ElasticQuerySpec extends ZIOSpecDefault {
           assert(queryWithValidationMethod.toJson(fieldPath = None))(equalTo(expectedWithValidationMethod.toJson)) &&
           assert(queryWithAllParams.toJson(fieldPath = None))(equalTo(expectedWithAllParams.toJson))
         },
+        test("geoShapeInline") {
+          val query =
+            geoShapeInline("testField", GeoPoint(20.0, 21.1))
+          val queryWithGeoPoint =
+            geoShapeInline(TestDocument.locationField, GeoPoint(20.0, 21.1)).relation(SpatialRelation.Intersects)
+          val queryWithGeoLine =
```

Comment:
These tests are for encoding, you are missing tests in `constructing` suite, which is first part of test class.